frameworks/PHP/symfony/benchmark_config.json

```diff
@@ -43,6 +43,29 @@
       "display_name": "symfony",
       "notes": "",
       "versus": "php"
+    },
+    "swoole": {
+      "plaintext_url": "/plaintext",
+      "json_url": "/json",
+      "db_url": "/db",
```

Comment:
Could be `/raw/db` for better performance.
Should we provide version that just provide swoole? Or a version that provide better performances with less features?

Reply:
I'll keep it as is for now,
The implementation with an ORM is more realistic with Symfony.
we'll see how it goes.